tests/integration/test_vulnerability_detector/test_scan_results/data/test_cases/cases_scan_updated_package_still_vulnerable.yaml

```diff
@@ -0,0 +1,14 @@
+- name: no_alert_updated_package_still_vulnerable
```

Comment:
Change the name to a shorter one with capital letters. I recommend: `VULN_REMAINS_IN_PACKAGE` or `PACKAGE_STILL_VULNERABLE`

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/4389/commits/18fa7b86f2c5959cfc8fd859de82e4f5ad17bd2a